src/core/stdc/errno.d

```diff
@@ -1830,6 +1835,109 @@ else version( OpenBSD )
     enum ENOTSUP            = 91;       /// Not supported
     enum ELAST              = 91;       /// Must be equal largest errno
 }
+else version( DragonFlyBSD )
+{
+    enum EPERM              = 1;        /// Operation not permitted
```

Comment:
I'm guessing you didn't write these comments yourself but copied them from somewhere?  If so, take out all the comments that you copied, see the linux block for an example.  We don't really want to deal with copyright issues.

Reply:
WIll do !
Funny how to most 'copyright bothersome os-ses' (ie: windows and osx) do have similar comments :-) Might be a good idea, to to the same for them as well.